server/views/station.html

```diff
@@ -235,12 +235,16 @@ <h2 class="govuk-heading-m govuk-!-margin-top-6">How levels here could affect ne
       {% for band in model.thresholds %}
       <div class="defra-flood-impact-list__row {% if band.isLatest %} defra-flood-impact-list__row--current {% elif band.isExceeded %} defra-flood-impact-list__row--exceeded{% endif %}" {% if band.type === 'historical' %} data-toggle-list-display-item="impact" {% endif %}>
         <dt class="defra-flood-impact-list__key">
-          {{ band.level }}m
+          {% if not model.station.isCoastal and band.level <= 0 %}
+            0m
+          {% else %}
+            {{ band.level }}m
+          {% endif %}
```

Comment:
logic should be worked out in the model and just pass through the worked out value